defaults/user/student.json

```diff
@@ -0,0 +1,28 @@
+{
+    "governmentId": "governmentIdStudent",
+    "authUser": {
+      "username": "student",
+      "password": "student",
+      "role": "Student"
+    },
+    "user":{
+      "username": "student",
+      "enrollmentIdSecret": "",
+      "isActive": true,
+      "role": "Student",
+      "address": {
+          "street": "Gänseweg",
+          "houseNumber": "42a",
+          "zipCode": "13337",
+          "city": "Entenhausen",
+          "country": "Germany"
+      },
+      "firstName": "Stu",
```

Comment:
I would prefer actual names for marketing reasons, the tag student should be sufficient for identifying students.